ts/util/asyncLoad/esm.ts

```diff
@@ -31,7 +31,9 @@ let root = new URL(import.meta.url).href.replace(
 if (!mathjax.asyncLoad) {
   mathjax.asyncLoad = async (name: string) => {
     const file = name.charAt(0) === '.' ? new URL(name, root).pathname : name;
-    return import(file).then((result) => result?.default || result);
+    return import(file).then((result) =>
+      result.default === undefined ? result : result.default
```

Comment:
Can `result.default` be `null`? If so, do we want `null` returned here?
I think this can only happen if the file has an `export default null;` entry. But even then we will probably want to take the content of `result`. 

In case the above reasoning holds, `result.default ?? result` would be more concise.

Reply:
I've made the change, and agree with your analysis.